### DISCUSSION TITLE
Adding tables to descriptions

### DISCUSSION THREAD

**Discussion by joergmerkle1973** [jesper-raemaekers/python-polarion#162] [General]
Hi, i would like to add tables to descriptions of work items. 

I looked up in the Polarion repository browser in the workitem.xml how tables look like and added tables via html tags < table >, < tr > and < td > to the description text. After  setting the description by 'workitem.setDescription' it ends up in workitem.xml with all tags replaced '<' to '& lt ;' and '>' to '& gt ;'. 

Interestingly, tables are shown correctly but i wonder if this is the correct way to add tables or if there is any other API that i missed...

**Comment by dzama-peter** [comment]
Hi! In my experience, I query'd a work item with tables through the getWorkitem() method and made a template out of the material I have gotten. When I uploaded a work item like that it seemed to be working fine for me. I hope it works for you as well!

**Reply by joergmerkle1973** [reply]
Thanks for your hints!
I have tried 'your way' and basically it is the same content as the way i'm setting it. I just got confused looking at the workitem.xml afterwards. 
Anyways, it is ok as long as there is no other advice from someone else...
